### PR TITLE
Changed dotnet version number.

### PR DESCRIPTION
**Pull request by edwardskrod**

### CODE REVIEW

File: .github/workflows/dotnetcore.yml [hunk 1]
```diff
@@ -5,13 +5,13 @@ on: [push]
...
     - uses: actions/checkout@v1
     - name: Setup .NET Core
       uses: actions/setup-dotnet@v1
       with:
-        dotnet-version: 2.2.108
-    - name: Build with dotnet
-      run: dotnet build --configuration Release
+        dotnet-version: '3.0.100'
+    - name: Build MyWpfApp with dotnet
+      run: dotnet build MyWpfApp\MyWpfApp.csproj --configuration Release
```
Comment (edwardskrod) [inline .github/workflows/dotnetcore.yml]: This is the command to actually build the project.

run: dotnet build MyWpfApp\MyWpfApp.csproj --configuration Release

File: .github/workflows/dotnetcore.yml [hunk 2]
```diff
@@ -5,13 +5,13 @@ on: [push]
 jobs:
   build:
 
-    runs-on: ubuntu-latest
+    runs-on: windows-latest
```
Comment (edwardskrod) [inline .github/workflows/dotnetcore.yml]: We had to change from ubuntu to windows in order to build the wpf app.

File: .github/workflows/dotnetcore.yml [hunk 3]
```diff
@@ -5,13 +5,13 @@ on: [push]
...
 
     steps:
     - uses: actions/checkout@v1
     - name: Setup .NET Core
       uses: actions/setup-dotnet@v1
       with:
-        dotnet-version: 2.2.108
-    - name: Build with dotnet
-      run: dotnet build --configuration Release
+        dotnet-version: '3.0.100'
```
Comment (edwardskrod) [inline .github/workflows/dotnetcore.yml]: You have to include the ENTIRE version number.  3.0.1 didn't work.

File: .github/workflows/dotnetcore.yml (hunk 1 above)
Comment (rconard) [reply .github/workflows/dotnetcore.yml]: 👍

**Pull request merged**